smalltalksrc/VMMaker-Tools/VMTrace.class.st

```diff
@@ -0,0 +1,58 @@
+Class {
+	#name : #VMTrace,
+	#superclass : #Object,
+	#instVars : [
+		'messageNodes'
+	],
+	#category : #'VMMaker-Tools'
+}
```

Comment:
This class was used for execution trace analysis. Not related to this work.

I'll remove it (but save it in some place before)